tests/kvmtest.sh

```diff
@@ -154,7 +155,8 @@ test_t1_lag() {
     lldp/test_lldp.py \
     route/test_default_route.py \
     platform_tests/test_cpu_memory_usage.py \
-    bgp/test_bgpmon.py"
+    bgp/test_bgpmon.py \
+    monitoring_processes/test_monitoring_critical_processes.py"
```

Comment:
Suggest renaming the directory to `process_monitoring` and the file to test_critical_process_monitoring.py

Reply:
Updated!